Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# holy light uther stuff

This reverts commit afca855bf0d8a485819f41eea2e951cbf80f0e0f.


## Changelog:
- Added Uther to the Expedition Activity and made him a bit more involved (players only mechanic)
- Changed that thing.


## Developer changelog:
- Added this thing.
- Changed that thing.


## Tests:


# How to test:
in the expedition event:
- play as arthas, do all 3 choices, make sure activity doesnt break. you should not get an event popup about changing your decision or whatever.
- play as jaina, make sure activity doesnt break
- play as uther, do all 3 arthas choices (using game rule), if arthas does not want to plague, you should not get an option to convince hiim otherwise. if you end up winning the battle to convince him (you can temporarily change the code to make it guaranteed, i added rng so its different everytime.), he should actually change the course of action and not just go along with the purge (unless he betrays you, which will be shown)
- make sure you can find uther to join the activity whether he is landed or not. will most likely find him if landed but test when he is unlanded
- do whatever else you think seems appropriate

# note:
if arthas decides not to purge the city, but his original intent was to go to northrend, he _should_ still be going to northrend regardless. idk if i should change his entire course of action, but i decided not to initally because its more likely to break the event, and mess up player choice
